lib/controller.js

```diff
@@ -180,26 +188,21 @@ module.exports = function createController(config) {
 
   function logoutCallback(req, res) {
     const serviceName = req.params.service;
-    const state = req.query.state;
-    const logoutState = req.session[`${serviceName}_logoutKey`];
-
-    if (state !== logoutState) {
```

Comment:
Has this become irrelevant?

Reply:
For now it is. It can be added later if we need it.